src/main/java/com/project/ita5/user/User.java

```diff
@@ -0,0 +1,42 @@
+package com.project.ita5.user;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+
+import javax.validation.constraints.NotBlank;
+
+
+public class User {
+    @Transient
+    public static final String SEQUENCE_NAME = "users_sequence";
+
+    @Id
+    private String id;
+
+    @NotBlank
+    private String username;
+
+    @NotBlank
+    private String password;
+
+    public User() {
+    }
+
+    public User(@NotBlank String username, @NotBlank String password) {
+        this.username = username;
+        this.password = password;
+    }
+
```

Comment:
a lot of empty lines. Use Intellij formatter.

Reply:
now fixed